src/posix/platform/udp.hpp

```diff
@@ -39,12 +41,13 @@ class Udp : public Mainloop::Source, private NonCopyable
 public:
     static Udp &Get(void);
 
-    void Init(const char *aIfName);
-    void SetUp(void);
+    void SetUp(otInstance *aInstance);
```

Comment:
Add documentation for public methods?

Reply:
I changed to define `SetUp()` and `TearDown()` as base methods in the 3rd commit of this commit